src/styles/ListOfBooks.css

```diff
@@ -10,7 +10,17 @@
     width: 25%;
 }
 
-.books-img {
+.valid-books-img {
     width: 200px;
     height: 300px;
 }
+
+.not-valid-books-img {
```

Comment:
You can combine CSS classes, so rather than pre-mixing your styles, have a `books-img` and a `disabled` class.